android/modules/android/src/java/ti/modules/titanium/android/AndroidModule.java

```diff
@@ -612,12 +604,7 @@ public void startService(IntentProxy intentProxy)
 	@Kroll.method
 	public void stopService(IntentProxy intentProxy)
 	{
-		TiApplication app = TiApplication.getInstance();
-		if (app != null) {
-			app.stopService(intentProxy.getIntent());
-		} else {
-			Log.w(TAG, "Application instance no longer available. Unable to stopService.");
-		}
+		TiApplication.getInstance().stopService(intentProxy.getIntent());
```

Comment:
This seem to be incorrectly solved merged conflicts. Please verify this for each file.

Reply:
This file was not in list of conflict files which I resolved. There were 3 iOS files only. Thanks!

Reply:
This part was merged correctly. I changed the code to this last week.
But thanks for double checking.

_**Side Note:** The `TiApplication` instance is guaranteed to exist for the lifetime of the app, which means that there's no need to do a `null` check. So, I simplified the code a bit._